.github/workflows/ci.yml

```diff
@@ -37,12 +37,15 @@ jobs:
       - name: Tests
         run: |
           /bin/bash -c " \
-            source ${SPACK_MANAGER}/start.sh && \
-            quick-create -n ci -s exawind@master && \
-            spack cd -e && \
+            source ${EXAWIND_MANAGER}/start.sh && \
+            spack-start && \
             ln -s ${GITHUB_WORKSPACE} exawind && \
-            spack develop exawind@master && \
-            spack manager external --latest --blacklist gmake exawind
+            quick-create-dev -s exawind@master && \
+            spack cd -e && \
```

Comment:
this line is not necessary. Running without `-n` or `-d` creates the env in the cwd
```suggestion
```